modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-store-info.adoc

```diff
@@ -150,13 +150,15 @@ Output:
 ----
 ====
 
+[[neo4j-admin-store-entity-limits]]
```

Comment:
We should move this section to **Store formats** now that we have a dedicated page. I can take care of it if you agree.